vendor/knative.dev/operator/pkg/reconciler/knativeserving/knativeserving.go

```diff
@@ -117,6 +117,10 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, ks *v1beta1.KnativeServi
 		security.AppendTargetSecurity,
 		common.AppendAdditionalManifests,
 		r.appendExtensionManifests,
+		func(ctx context.Context, manifest *mf.Manifest, component base.KComponent) error {
+			*manifest = manifest.Filter(mf.Not(mf.All(mf.ByKind("Namespace"), mf.ByName("kourier-system"))))
```

Comment:
Which manifests are the ones filtered out? And why is it needed now (AFAIK we only add knative-serving-ingress, right?)

Reply:
We have [logic](https://github.com/openshift-knative/serverless-operator/blob/main/openshift-knative-operator/pkg/serving/kourier.go#L66) that inject the ns in a resource. Whatever has the provider label set to kourier will be transformed using the ns inject [here](https://github.com/openshift-knative/serverless-operator/blob/main/openshift-knative-operator/pkg/serving/kourier.go#L67) including a [namespace](https://github.com/openshift-knative/serverless-operator/blob/main/vendor/github.com/manifestival/manifestival/transform.go#L63).